ui/pages/send/send-content/send-gas-row/send-gas-row.component.js

```diff
@@ -40,7 +46,11 @@ export default class SendGasRow extends Component {
             category: 'Transactions',
             event: 'Clicked "Advanced Options"',
           });
-          showCustomizeGasModal();
+          if (isEIP1559Network) {
```

Comment:
I don't think we should show the customization modal in send when on an EIP-1559 network

Reply:
This code can't be hit, but it should be removed